examples/symfony.php

```diff
@@ -45,6 +46,13 @@ public function json()
 
         return print_r($payload, true);
     }
+
+    public function valuesArray()
+    {
+        $name = $this->queryParameter('arr')->commaSeparated()->int();
```

Comment:
What happens in case no `arr` parameter is provided (e.g. calling `symfony.php?action=valuesArray` instead of `symfony.php?action=valuesArray&arr=1`)?

I guess here we should also provide a `required` and `defaultsTo`, e.g. `$this->queryParameter('arr')->commaSeparated()->int()->required()` would throw an exception in case `symfony.php?action=valuesArray` is called because no `arr` parameter is given. While `$this->queryParameter('arr')->commaSeparated()->int()->defaultsTo([])` would just return an empty list.